app/src/main/java/ch/epfl/culturequest/ui/map/MapFragment.java

```diff
@@ -0,0 +1,37 @@
+package ch.epfl.culturequest.ui.map;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProvider;
+
+import ch.epfl.culturequest.databinding.FragmentMapBinding;
```

Comment:
Just to understand what is a Binding in this situation?

Reply:
Binding is a support library that allows you to bind UI components in your layouts to data sources in your app using a declarative format rather than programmatically, i.e instead of using findViewById(). Why is it better ? Because it allows to observe data changes easily and to update Views on the fly while also detecting Null Pointer exceptions during build time instead of runtime which is better for testing.